vello_encoding/src/encoding.rs

```diff
@@ -449,10 +446,11 @@ impl Encoding {
             .extend_from_slice(bytemuck::cast_slice(bytemuck::bytes_of(&DrawImage {
                 xy: 0,
                 width_height: (brush.image.width << 16) | (brush.image.height & 0xFFFF),
-                sample_alpha: ((quality as u32) << 12)
+                sample_alpha: ((brush.image.format as u32) << 13
```

Comment:
Should this be `14`, since `ImageQuality` has three variants?

Reply:
Hm, I think it should. (Un)packing code feels unsafe, as there are some assumptions of types that can change. We should at least have some const asserts here for size assumptions (by looking using https://doc.rust-lang.org/std/primitive.u32.html#method.leading_zeros on max of types).